lib/apiservers/engine/backends/container.go

```diff
@@ -151,7 +151,7 @@ func (c *Container) ContainerCreate(config types.ContainerCreateConfig) (types.C
 	var layer *viccontainer.VicContainer
 	container := viccontainer.GetCache().GetContainerByName(config.Config.Image)
 	if container == nil {
-		layer, err = c.getImageMetadataFromStoragePL(config.Config.Image)
+		layer, err = c.getImageMetadataFromImageC(config.Config.Image)
```

Comment:
If you're using this just to get the layer ID, it's in `ImageConfig.V1Image.ID`.